Task_4/task_4.py

```diff
@@ -0,0 +1,43 @@
+# Multi paradigm programming language
```

Comment:
I know it's confusing but you need to find some balance between extremely small functions with atomary action and combining all the stuff in the same function. Let's try to discuss your code during the class